code/script.js

```diff
@@ -0,0 +1,68 @@
+const USER = "AschwinSiaila";
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`;
+const nameAndPicture = document.getElementById("nameandpicture");
+const projectsContainer = document.getElementById("projects");
+
+const NameandPicture = () => {
+  const personal_ID = `https://api.github.com/users/${USER}`;
+  fetch(personal_ID)
+    .then((res) => res.json())
+    .then((data) => {
+      global__UserData = data;
+    })
+    .then(() => NameAndPicture());
+};
+
+const NameAndPicture = () => {
+  const picture = global__UserData.avatar_url;
+
+  nameAndPicture.innerHTML += `
+        <img class="profile-pic" src="${picture}" alt="Picture of gitHub-user"/>
+        <div class="profile-name">${USER}</div>
+      `;
+};
```

Comment:
These functions have the same name. In the first one you fetching the data and another one you creating HTML. If you combine those two you can have only one function and have mu less lines of code but the same functionality :)  just copy everything that you have in your second function and put it in your first one after line 13